src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstSide;
+    private int secondSide;
+    private int height;
+
+    public IsoscelesTrapezoid(int firstSide, int secondSide, int height, Color color) {

```

Comment:
The superclass constructor should be called before assigning local variables. Instead of assigning the color directly, it should be passed to the superclass constructor.

Reply:
Done it.